src/page-header/index.md

```diff
@@ -0,0 +1,27 @@
+# What is a page header?
+
+Headers are the big, banner-like images that adorn the tops of game pages. They are optional but highly, highly recommended. Headers make for a splendid looking page.
+
+You might want to experiment with different images to see what looks good as a header. Screenshots often turn out well, but some of the nicest looking headers are created for the purpose. 
+
+## What are the header requirements and guidelines?
+
+Here are the requirements and recommendations for header images:
+
+- The image format can be PNG, JPG, or GIF. PNGs will result in the highest quality. 
+- In general, your image should be as large as possible. 
+- The minimum recommended width is 1920px. Images less wide than this will be automatically enlarged. The maximum width is 5120px.
+- The minimum ratio (width/height) is 2.7 and the maximum ratio is 6.3. The recommended ratio is 4. 
+	- We recommend making the width 400% of the height (it has to be between 271.5% and 630%). 
```

Comment:
Can we swap this to something like "We recommend making the height 25% of the width" ? The three digit percentages are a bit scary to look at.